ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -570,6 +570,53 @@ def Conv3D(
     )
 
 
+@to_ivy_arrays_and_back
+def Conv2D(
+    *,
+    input,
+    filter,
+    strides,
+    padding,
+    use_cudnn_on_gpu=False,
+    data_format='NHWC',
+    dilations=[1, 1, 1, 1],
+    name="Conv2D"
+
+):
+    # ivy.backends.tensorflow expects strides and dilations to be
+    # a single integer value or a list of 2 values whereas the raw op
+    # expects a list of 4 values
+    if data_format == "NHWC":
+        strides = strides[1:-1]
+        dilations = dilations[1:-1]
+    elif data_format == "NCHW":
+        strides = strides[2:]
+        dilations = dilations[2:]
+
+    # for Conv2D, the explicit_padding is defined as
+    # [[0, 0], [pad_top, pad_bottom], [pad_left, pad_right], [0, 0]]
+    # when the data_format is "NHWC"
+    # for Conv2D, the explicit_paddings is defined as
+    # [[0, 0], [0, 0], [pad_top, pad_bottom], [pad_left, pad_right],
+    # when the data_format is "NCHW"
+    if padding == "EXPLICIT" and data_format == "NHWC":
+        padding = padding[1:2]
```

Comment:
And here it should be `padding = explicit_paddings[1:2]`. Same below.

Reply:
@AnnaTz  but it should return it to the padding variable only right

Reply:
I'm not sure what you mean.

Reply:
check the current commit

Reply:
@AnnaTz I think this should work, as the function `Conv2D` can take the `explicit_paddings` and return the padding values for explicit_padding to `ivy.conv2d` function in the same `padding `variable that is defined for `ivy.conv2d` function.

Reply:
It seems alright now

Reply:
ok so, now can we merge it.

Reply:
The tests need to be passing to merge it. Even though it seems ok, there might still be some bugs we're not seeing.

Reply:
i am running the tests locally.